ovisbot/extensions/ctf/ctf.py

```diff
@@ -954,7 +965,49 @@ async def check_reminders(self):
                     )
             except CTF.DoesNotExist:
                 continue
-
+    
+    # TODO Make async somehow to avoid request stall
+    def pushToGitHub(self,ctf_name,challenge_name):
+
+        pinsDir = os.path.abspath(os.getcwd()) + '/pins/'
```

Comment:
Use format strings (same for everywhere else)
```suggestion
        pinsDir = f'{os.path.abspath(os.getcwd())}/pins/'
```

Reply:
fixed, thanks!